src/citrine/resources/project.py

```diff
@@ -315,6 +315,71 @@ def make_private(self,
         })
         return True
 
+    def creator(self) -> str:
+        """
+        Return the creator of this project.
+
+        Returns
+        -------
+        str
+            The email of the creator of this resource.
+
+        """
+        email = self.session.get_resource(self._path() + "/creator")["email"]
+        return email
+
+    def owned_dataset_ids(self) -> List[str]:
+        """
+        List all the ids of the datasets owned by the current project.
+
+        Returns
+        -------
+        List[str]
+            The ids of the modules owned by current project
+
+        """
+        dataset_ids = self.session.get_resource(self._path() + "/dataset_ids")["dataset_ids"]
+        return dataset_ids
+
+    def owned_table_ids(self) -> List[str]:
+        """
+        List all the ids of the tables owned by the current project.
+
+        Returns
+        -------
+        List[str]
+            The ids of the tables owned by current project
+
+        """
+        table_ids = self.session.get_resource(self._path() + "/table_ids")["table_ids"]
+        return table_ids
+
+    def owned_table_config_ids(self) -> List[str]:
+        """
+        List all the ids of the table configs owned by the current project.
+
+        Returns
+        -------
+        List[str]
+            The ids of the table configs owned by current project
+
+        """
+        result = self.session.get_resource(self._path() + "/table_definition_ids")
+        return result["table_definition_ids"]
+
+    def owned_module_ids(self) -> List[str]:
```

Comment:
I don't think we need to expose this since modules cannot currently be shared, and I don't think we want to since we're generally moving to more strongly typed modules (e.g. differentiating between predictors and design spaces).

Reply:
Makes sense to me, and they'll get pulled once it's clear what ESI's opinions are.

Presumably this means you agree with just listing UIDs instead of building objects.

Reply:
@maxhutch LGTM. Trusting test coverage as our safety net.